script/travis_setup

```diff
@@ -15,6 +15,13 @@ else
     echo database.yml exists
 fi
 
+if [ ! -f config/gmaps_api_key.yml ]; then
```

Comment:
We need to make sure this file exists because later Travis CI throws an error if it's missing. 
See, e.g., https://travis-ci.org/github/mushroomobserver/mushroom-observer/builds/679543364